databox/api/src/Asset/Attribute/InitialAttributeValuesResolver.php

```diff
@@ -131,7 +132,11 @@ private function resolveInitial(
         // remove empty values
         return array_filter(
             $initialValues,
-            function (string $s): bool {
+            function (?string $s): bool {
+                if (null === $s) {
```

Comment:
if we return "" as value for blank tag, we _may_ not need to test null.
But let's keep this test to protect `trim()` not accepting null anymore.